shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/BaseRule.g4

```diff
@@ -183,6 +183,10 @@ accessDriverType
     : identifier
     ;
 
+partition
+    : identifier
+    ;
+
```

Comment:
please use `partitionName` instead of `partition`

Reply:
ok, this `partition` is not use, i will remove it.